src/v2/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -163,7 +163,18 @@ export class FollowArtistButton extends React.Component<Props, State> {
 
     // Custom button renderer
     const content = render ? (
-      <span onClick={this.handleFollow}> {render(artist)}</span>
+      <span
+        tabIndex={0}
+        onKeyPress={event => {
+          if (event.key === "Enter" || event.key === " ") {
+            this.handleFollow
+          }
+        }}
+        onClick={this.handleFollow}
```

Comment:
Ah one more thing: can you give these a `role="button"` (which is one of the distinctions that iOS voiceover uses)

Reply:
Updated!